dynawo/sources/Models/Modelica/Dynawo/Electrical/Shunt.mo

```diff
@@ -0,0 +1,43 @@
+within Dynawo.Electrical;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
```

Comment:
As you will have to modify it a bit anyway (due to the year change), I also decided (I haven't yet warned everyone in the team) to modify the text to say:

This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems

Reply:
done ;)